BinaryTree.java

```diff
@@ -0,0 +1,68 @@
+package consistent;
+
+/**
+ * Created by deepakrtp on 01/04/17.
+ */
+import io.netty.bootstrap.Bootstrap;
+
+
+public class BinaryTree {
+
+    public final int NOOFSERVERS = 8;
```

Comment:
Also what you really want to define a constant is:
private static final int NO_OF_SERVERS = 7; (added a static extra there, please understand what difference that it makes)

(Look at how I added underscores in the naming of NO_OF_SERVERS, this is a java standard for constant variables.)